src/components/RepoItem.vue

```diff
@@ -29,7 +29,7 @@
       <div v-if="build" class="build">
         <img :src="avatar" alt="avatar"/>
         <div class="description">
-          <span>{{build.author_login}}</span>
+          <span>{{actor}}</span>
```

Comment:
It isn't clear to me why we should to create `getBuildActor` and pass its result as `actor` prop. I think it's simpler and clearer to write
```
{{build.event === "promote" ? build.trigger : build.author_login}}
```
right in the RepoItem.
I am sure that it's okay to place `if` right in the `RepoItem` because it's a common case, not a special logic, and, for example, we do so in the `RepoItemLabel`

Reply:
My thinking was that this bit of code is used in a few components, so if we ever had to update the logic to get the actor, it'd be easier and less error-prone to just change this one function. Also it's probably a style preference, but imo the template code is simpler if it just reads `actor` since the prop name documents its contents. As someone reading the template, I can quickly visualize that the contents of the component are actor -> action -> link. Finally, to me `getBuildActor` felt similar to the `isBuildFinished` function in `buildHelper.js,` so I added it there.

This is not a super strong opinion and I'm happy to change it -- just wanted to share my thought process. Lmk!